### ISSUE TITLE
Disable collision after setCollisionMask

### ISSUE THREAD

**Issue by FuriouZz** [saharan/OimoPhysics#6]
Hello!

I got a sphere, a box and a cylinder. I want that when my sphere rolls on the box but when it touches the cylinder, the sphere falls.

I update collisionGroup/collisionMask when I have a contact between the sphere and the cylinder, but it continues collide with the cylinder and with my box.

Sample code
```js
// Collision Group
var CollisionGroup = {
  FLOOR: 1,
  SPHERE: 2,
  SPHERE_OUT: 4,
  BOX: 8,
  CYLINDER: 16
}

// Sphere
sphereShape.addCollisionGroup( CollisionGroup.SPHERE );
sphereShape.addCollisionMask( CollisionGroup.FLOOR );

// Box
boxShape.addCollisionGroup( CollisionGroup.BOX );
boxShape.addCollisionMask( CollisionGroup.BALL );

// Cylinder
cylinderShape.addCollisionGroup( CollisionGroup.CYLINDER );
cylinderShape.addCollisionMask( CollisionGroup.SPHERE );

// Functions
function addSphere( sphere ) {
  sphereShape.setCollisionGroup( CollisionGroup.SPHERE );
  sphereShape.setCollisionMask( sphereShape.getCollisionMask() | CollisionGroup.CYLINDER | CollisionGroup.BOX );
}

function removeSphere( sphere ) {
  sphereShape.setCollisionGroup( CollisionGroup.SPHERE_OUT );
  sphereShape.setCollisionMask( CollisionGroup.FLOOR );
}
```

**Comment by saharan** [comment]
Hello!

I see, that's my mistake; I'll fix it in the next release.
Thank you for detail reporting :)

**Comment by saharan** [comment]
This bug has been fixed in v1.2.0
https://github.com/saharan/OimoPhysics/commit/a4f307b05eb87d3b1d8a86293cdf75ede7f7b92b !
(please reopen if there are still problems)
